src/main/resources/io/jenkins/plugins/thememanager/ThemeManagerPageDecorator/global.jelly

```diff
@@ -1,6 +1,9 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler" xmlns:th="/lib/theme-manager">
-  <f:section title="${%Built-in Themes}">
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:l="/lib/layout" xmlns:th="/lib/theme-manager">
+  <f:section title="${%Themes}">
```

Comment:
the reason this was called Built-in Themes was to stop the double themes label

![image](https://user-images.githubusercontent.com/21194782/205017877-e3f74bbf-3854-48c7-a7d4-d49894ccacb7.png)

You could probably fix that by making the radio label visually hidden or call it something like select a theme

Reply:
Done, changed it to 'Select theme' 👍